libraries/aws_ec2_eip.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsEc2Eip < AwsResourceBase
+  name 'aws_ec2_eip'
+  desc 'Specifies an Elastic IP (EIP) address and can, optionally, associate it with an Amazon EC2 instance.'
+
+  example "
+    describe aws_ec2_eip(public_ip: '10.1.1.10') do
+      it { should eq '10.1.1.10' }
+    end
+
+    describe aws_ec2_eip(public_ip: '10.1.1.10') do
+      it { should exits }
+    end
+  "
```

Comment:
add one more example for this, 

```
describe aws_ec2_eip('10.1.1.10') do
      it { should exits }
    end
```

Reply:
??

Reply:
Already examples are there in the documentation.

Reply:
@sathish-progress  if someone wants to see the examples then one should go to the documentation and see how it works. Right?

Reply:
Done